rest-api-spec/src/main/resources/rest-api-spec/test/indices.put_alias/10_basic.yml

```diff
@@ -76,3 +76,30 @@
         index: test_index
         name: test_alias
   - match: {test_index.aliases.test_alias: { 'is_write_index': true }}
+
+---
+"Can create filtered alias with a date_nanos range query":
+  - skip:
+      version: " - 6.8.99"
```

Comment:
I think this needs changing to " - 7.99.99" initially since the bugfix will land on the master branch first. We also run this tests in "mixed" cluster scenarios to simulate behaviours during a rolling upgrade, so this might hit a 7.latest/8.0 cluster and would then could fail if hitting a 7.x node without the fix first.
The way we usually handle this is disabeling the tests except for the version this is first checked in, then backport and adapt the versions accordingly. I can handle that if you want, probably faster than going through extra PRs...